<?xml version="1.0"?>
<launch>
  <node pkg="point_cloud_features" type="point_cloud_features" name="point_cloud_features" output="screen" respawn="false">
    <param name="minimum_points"         type="int"  value="5"/>
    <param name="number_of_points"       type="bool" value="true"/>
    <param name="min_distance"           type="bool" value="true"/>
    <param name="covariance_mat3D"       type="bool" value="true"/>
    <param name="normalized_MOIT"        type="bool" value="true"/>
    <param name="slice_feature"          type="bool" value="true"/>
    <param name="intensity_distribution" type="bool" value="true"/>
  </node>
</launch> 
